packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/detail.controller.js

```diff
@@ -0,0 +1,237 @@
+import assignIn from 'lodash/assignIn';
+import filter from 'lodash/filter';
+import isObject from 'lodash/isObject';
+import map from 'lodash/map';
+import snakeCase from 'lodash/snakeCase';
+
+export default class PackHostedEmailDetailCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $state,
```

Comment:
Could all concerns related to ui router be extracted to the state declaration ?

Reply:
Yes, it could be. But, for the moment, I haven't refactored the parent (which need more time than just replace old WS calls with API calls). I note this for a future refactoring task with ui router to replace those state declarations. Are you agree with that @marie-j  ?